distributed/scheduler.py

```diff
@@ -3571,6 +3571,7 @@ def __init__(
         else:
             self.idle_timeout = None
         self.idle_since = time()
+        self.no_worker_timeout = None
```

Comment:
updated to self.no_worker_since = time() to remain consistent with self.idle_since, this will still be set to None when workers are processing tasks.